certificate_path.go

```diff
@@ -0,0 +1,6 @@
+package globalsign
+
+// CertificatePathResponse .
```

Comment:
This comment is not really appropriate, it should be a full sentence, go doc style.
Is it necessary to export this? I.e. is it used outside?

Reply:
The point of making request and response struct exported is because, we make client API also, so users can do the API request to GlobalSign service if needed.